src/rockstor/storageadmin/static/storageadmin/js/templates/dashboard/widgets/pool_usage.jst

```diff
@@ -1,38 +1,35 @@
 <script>
 /*
- * Copyright (c) 2016 RockStor, Inc. <http://rockstor.com>
+ * Copyright (c) 2012-2016 RockStor, Inc. <http://rockstor.com>
  * This file is part of RockStor.
- *
+ * 
  * RockStor is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published
  * by the Free Software Foundation; either version 2 of the License,
  * or (at your option) any later version.
- *
+ * 
  * RockStor is distributed in the hope that it will be useful, but
  * WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
  * General Public License for more details.
- *
+ * 
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
- *
+ * 
  */
 </script>
 
-<div class="row">
```

Comment:
No! Thanks, going to check on other Shares Widget PR too.

Solved on f26a776 :smiley: